circe/src/main/scala/org/http4s/circe/package.scala

```diff
@@ -15,5 +15,9 @@
  */
 
 package org.http4s
+package circe
 
-package object circe extends CirceInstances
+import org.typelevel.scalaccompat.annotation._
+
+@nowarn213("msg=package object inheritance is deprecated")
+object `package` extends CirceInstances
```

Comment:
Thanks for this workaround, I stole and applied it in several typelevel projects already.

Reply:
Thanks for your help upgrading builds :pray: